user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -716,10 +719,14 @@ define([
           obj.file = response.children[i].file;
           obj.file.pathText = jQuery.i18n.prop('originText') + " " //i18n
 
+          if( isRootPath(obj.file.path) ){
+            obj.file.isRootPath = true;
+          }
+
           //decode potentially encoded name and title attributes of PVFS paths
           if( !isRepositoryPath(obj.file.path) && !obj.file.decoded ){
             obj.file.name = this.decodePvfsFileAttribute(obj.file.name);
-            obj.file.title = obj.file.title === null ? null : this.decodePvfsFileAttribute(obj.file.title);
+            obj.file.title = obj.file.title === null ? obj.file.name : this.decodePvfsFileAttribute(obj.file.title);
```

Comment:
Does this belong to this PR? Is it for BACKLOG-41091?

Reply:
Related to https://github.com/pentaho/pentaho-platform/pull/5638#discussion_r1644574159, I will move this to another PR